### ISSUE TITLE
Settings

### ISSUE THREAD

**Issue by pares49** [Jamee999/cricket#32]
Sorry for these questions - I am a bit new to python.

How does one make use of the settings text file? Do you need to edit the file to say "active=true"?

Do you have to set the pace, spin, and outfield settings? Or will these be set randomly between 0.5 and 1.5? 

If you set the venue as eg. MCG - will the spin, pace, and outfield settings adjust to reflect this?

How do you set it to see the bowling changes? What input is needed

**Comment by Jamee999** [comment]
Yeah, you need to edit the text file. If the top line is not "active = true", nothing will happen.

Any of the settings that are marked as "default" the sim will precede as normal. So if you have a pitch setting as default, it will auto-generate values

The ground option is purely cosmetic, it doesn't change the pitch, it just shows up on the scorecard.

If you change "showbowlingchanges" to "true" it should work, as long as the top line is also "true"

**Comment by pares49** [comment]
Cheers Jamee - v cool programme
